app/views/smart_answers/result.html.erb

```diff
@@ -23,15 +21,18 @@
 
     <div class="govuk-!-margin-bottom-6 result-body">
       <% if outcome.title.present? %>
-        <%= render "govuk_publishing_components/components/heading", {
-          text: outcome.title,
-          margin_bottom: 6
-        } unless outcome.title_as_heading? %>
+        <%= render "govuk_publishing_components/components/title", {
+        context: outcome.heading_title,
+        title: outcome.title,
+        margin_bottom: 5
```

Comment:
There's a `margin_bottom` on this side of the if statement - does the other title component on line 30-32 need that margin as well?

Reply:
🤔 Taking a second look I've removed both as it keeps the defaults (which is `margin_bottom: 8`).